### DISCUSSION TITLE
Blink+Build R & typescript for AWS CDK

### DISCUSSION THREAD

**Discussion by patoscope** [blinksh/blink#2068] [Q&A]
Hi!

Do you support build environnement for coding in R and also coding AWS CDK in typescript?

I would like to know if that is at all possible before signing up.

Cheers!

**Comment by carloscabanero** [comment]
We do support R and Typescript, so AWS CDK should work properly although I haven't done any development with it myself. There is a free trial of Build, but let me know if you need longer time or anything and I can set it up for you. Also, you can run any container, so if you have your own toolset ready to go, you could potentially do that too.

**Reply by patoscope** [reply]
That’s awesome!  I’ll get on it!  Thanks for the response.
